scripts/drone/steps/lib.star

```diff
@@ -633,8 +633,8 @@ def frontend_metrics_step(trigger = None):
         },
         "failure": "ignore",
         "commands": [
-            "apk add --update bash",
-            "./scripts/ci-frontend-metrics.sh | ./bin/build publish-metrics $${GRAFANA_MISC_STATS_API_KEY}",
```

Comment:
Not sure removing { } here will make any difference.

Reply:
it does with `sh`.

Reply:
Maybe it would be better to have it between quotes for example `"$${GRAFANA_MISC_STATS_API_KEY}"`

Reply:
Oh, I see with `sh` it makes sense.

Reply:
but upgrading grep definitely did fix the issue I was having.